app/src/main/java/org/wikipedia/places/PlacesFragment.kt

```diff
@@ -88,20 +95,48 @@ class PlacesFragment : Fragment(), LinkPreviewDialog.Callback, MapboxMap.OnMapCl
     private val markerRect = Rect(0, 0, MARKER_WIDTH, MARKER_HEIGHT)
     private val markerPaintSrc = Paint().apply { isAntiAlias = true; xfermode = PorterDuffXfermode(Mode.SRC) }
     private val markerPaintSrcIn = Paint().apply { isAntiAlias = true; xfermode = PorterDuffXfermode(Mode.SRC_IN) }
+    private var searchRadius: Int = 50
+    private var zoom: Double = 15.0
+    private var magnifiedMarker: Symbol? = null
 
     private val locationPermissionRequest = registerForActivityResult(ActivityResultContracts.RequestMultiplePermissions()) { permissions ->
         when {
             permissions.getOrDefault(Manifest.permission.ACCESS_FINE_LOCATION, false) ||
             permissions.getOrDefault(Manifest.permission.ACCESS_COARSE_LOCATION, false) -> {
                 startLocationTracking()
-                goToLastKnownLocation(1000, viewModel.location, viewModel.pageTitle != null)
```

Comment:
@cooltey I removed this change as the zoom was not that much different between 15.99 and 15. Instead it now enlarges the icon. Robin has mentioned 2x in the specs but I have settled for 1.75. will discuss this during design review.